Makefile.js

```diff
@@ -810,21 +820,25 @@ target.gensite = function(prereleaseVersion) {
     echo("> Generating the rule listing (Step 11)");
     generateRuleIndexPage(process.cwd());
 
-    // 12. Delete temporary directory
-    echo("> Removing the temporary directory (Step 12)");
+    // 12. Generate team page data
+    echo("> Generating the team data (Step 12)");
+    generateTeamData();
```

Comment:
Could we add a check to skip this step if `ESLINT_GITHUB_TOKEN` is unavailable? It's useful for contributors to be able to run `npm run gensite` or `npm run generate-release` when testing changes out locally even if they're not organization members.

Reply:
Do you have a suggestion for how to allow that and also not silently fail on this step? The latter is my primary concern.

Another option: We could also run the `fetch-team-data.json` script out-of-band from gensite so that it only happens during an official release?

Reply:
Well, we could prevent the failure from being "silent" by printing a message, e.g.

> Did not fetch team data because no `ESLINT_GITHUB_TOKEN` environment variable was present (Step 12)

Running it out-of-band also sounds fine. A potential snag is that the process of cloning `eslint.github.io`, making changes to `eslint.github.io`, and making a local commit to `eslint.github.io` all happen as part of a local release. So the out-of-band script would presumably have to run afterwards and amend the commit.

A third option would be to separate this from the release process entirely, and just run the script periodically on the Jenkins server.

Reply:
We only need to make the updates whenever we modify the teams, right? Is there a specific reason it needs to be done every release?